docs/naps/0-nap-process.md

```diff
@@ -0,0 +1,298 @@
+
+For an equivalent example in the NumPy library, see: <https://mail.python.org/pipermail/numpy-discussion/2018-June/078345.html>
+
+After you write the post, you should make sure to link to the specific
+thread from the `Discussion` section of the NAP, so that people can
+find it later.
+
+Generally the NAP author will be the one to make this post, but
+anyone can do it – the important thing is to make sure that everyone
+knows when a NAP is on the verge of acceptance, and give them a final
+chance to respond. If there's some special reason to extend this final
+comment period beyond 7 days, then that's fine, just say so in the
+post. You shouldn't do less than 7 days, because sometimes people are
+traveling or similar and need some time to respond.
+
+In general, the goal is to make sure that the community has consensus,
+not provide a rigid policy for people to try to game. When in doubt,
+err on the side of asking for more feedback and looking for
+opportunities to compromise.
+
+If the final comment period passes without any substantive objections,
+then the NAP can officially be marked `Accepted`. You should send a
+followup post notifying the thread (celebratory emoji optional but
+encouraged 🎉✨), and then update the NAP by setting its `:Status:`
+to `Accepted`, and its `:Resolution:` header to a link to your
+followup post.
+
+If there *are* substantive objections, then the NAP remains in
+`Draft` state, discussion continues as normal, and it can be
+proposed for acceptance again later once the objections are resolved.
+
+In unusual cases, when no consensus can be reached between core developers,
+the [napari Steering Council] may be asked to decide whether a
+controversial NAP is `Accepted`.
+
+### Maintenance
+
+In general, Standards track NAPs are no longer modified after they have
+reached the Final state, as the code and project documentation are
+considered the ultimate reference for the implemented feature. They may,
+however, be updated under special circumstances.
+
+Process NAPs may be updated over time to reflect changes
+to development practices and other details. The precise process followed in
+these cases will depend on the nature and purpose of the NAP being updated.
+
+## Format and Template
+
+NAPs are UTF-8 encoded text files using the [MyST markdown] format.  Please
+see the {ref}`nap_template` file and the [MyST markdown cheat sheet] for
+more information.  We use [Sphinx] to convert NAPs to HTML for viewing on
+the web [^id4].
+
+### Header Preamble
+
+Each NAP must begin with a header preamble.  The headers
+must appear in the following order.  Headers marked with `*` are
+optional.  All other headers are required.
+
+```
+  :Author: <list of authors' real names and optionally, email addresses>
+  :Status: <Draft | Active | Accepted | Deferred | Rejected |
+           Withdrawn | Final | Superseded>
+  :Type: <Standards Track | Process>
+  :Created: <date created on, in dd-mmm-yyyy format>
```

Comment:
```suggestion
  :Created: <date created on, in yyyy-mm-dd format>
```

This format is not language dependent (i.e. `mmm` changes dependent on language whereas `mm` does not) and also sorts chronologically.

Reply:
Oh, I see we're actually using that format elsewhere, so I guess this one is just inconsistent.